fr/index.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
+<html lang="fr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Les mythes de l’accessibilité</title>
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="icon" type="image/x-icon" href="../favicon.ico" />
+
+    <meta name="description" content="A small project debunking common accessibility myths." />
+    <meta name="image" content="https://a11ymyths.com/" />
+
+    <meta property="og:title" content="Les mythes de l’accessibilité" />
+    <meta property="og:description" content="A small project debunking common accessibility myths." />
+    <meta property="og:image" content="https://a11ymyths.com/images/og-image.png" />
+    <meta property="og:image:alt" content="Les mythes de l’accessibilité" />
+    <meta property="og:url" content="https://a11ymyths.com/" />
+
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:creator" content="@_sergeikriger" />
+    <meta name="twitter:title" content="Les mythes de l’accessibilité" />
+    <meta name="twitter:description" content="A small project debunking common accessibility myths." />
```

Comment:
@MarionCouesnon Could you translate this phrase.

Reply:
Done with 646f61ff26a73ba84f69468cafc139a12f7baf24